docs/app/pages/components/components-sections/tables/filters/filters.component.html

```diff
@@ -17,21 +17,27 @@
     <tbody>
         <tr class="clickable" *ngFor="let document of documents">
             <td>
-                <div class="listview-page-icon"></div>
+                <ux-icon
+                    name="document"
+                    size="24px"
+                    [flipHorizontal]="false"
+                    [flipVertical]="false">
+                </ux-icon>
             </td>
             <td>{{ document.name }}</td>
             <td>{{ document.author }}</td>
             <td>{{ document.date }}</td>
             <td>
-                <ux-spark theme="accent"
+                <ux-spark [theme]="'ok'"
```

Comment:
String values should not use `[]` simply use `theme="ok"`. However in this case we have `chart1`, `chart2` etc.. it should be one of these colours that is used.

Reply:
Good point, `chart2` should be the same color in the Keppel theme.